web_view_transition/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Odoo, Open Source Web Transition
+# Copyright (C) 2019 Alexandre Díaz <dev@redneboa.es>
+#
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).#
+{
```

Comment:
Add `website` key and URL should be `https://github.com/OCA/web`

Reply:
done :)